src/components/AccordionItem/AccordionItem.js

```diff
@@ -140,14 +140,12 @@ export default class AccordionItem extends Component {
               className={`${prefix}--accordion__arrow`}
               icon={iconChevronRight}
               aria-label={iconDescription}
-              role={null} // eslint-disable-line jsx-a11y/aria-role
             />
           ) : (
             <Icon
               className={`${prefix}--accordion__arrow`}
               icon={iconChevronRight}
               description={iconDescription}
-              role={null} // eslint-disable-line jsx-a11y/aria-role
```

Comment:
Looking at @emyarod's earlier change at https://github.com/IBM/carbon-components-react/pull/1657#discussion_r241844161, we still need `role={null}` for stable icon here. @emyarod Is it correct?

Reply:
yes we still need `role={null}` for the legacy icon 

ref: https://github.com/IBM/carbon-components-react/pull/1657/files#r242300911

Reply:
@asudoh @emyarod Have added it back in, thanks!